include/fmt/format-inl.h

```diff
@@ -23,7 +23,7 @@
 #endif
 
 #ifdef _WIN32
-#  ifdef FMT_BIG_WIN
+#  if defined(FMT_BIG_WIN) || defined(__MINGW32__) || defined(__MINGW64__)
```

Comment:
The problem is not mingw per se but the fact that NOMINMAX is defined (+ Werror), so `|| defined(__MINGW32__) || defined(__MINGW64__)` should be replaced with `|| defined(NOMINMAX)`.

Reply:
Fixed!